spec/switchChannel.spec.js

```diff
@@ -85,10 +85,10 @@ describe.each(cases)("Switch Channel functionality - Consent: %s - iFrame: %s",
           expect(newSid).not.toBe(sid);
         });
 
-        it("should call /meta endpoint", async () => {
+        it("should call /meta.gif endpoint", async () => {
           const response = await metaCalled;
           expect(response.url()).toBe(
-            `http://localhost:3000/meta?sid=${newSid}&idtype=1&ccid=1&onid=1&nid=1&name=TEST&isHD=true`,
+            `http://localhost:3000/meta.gif?sid=${newSid}&idtype=1&ccid=1&onid=1&nid=1&name=TEST&isHD=true&spc=0&vid=undefined`,
```

Comment:
I dont think you should add a parameter if you do not have the value for a parameter here .. e.e. vid should not be set in this case